internal/orchestrator/bound_callbacks.go

```diff
@@ -30,22 +31,35 @@ type boundCallbacks struct {
 	dx dataexchange.Plugin
 	ss sharedstorage.Plugin
 	ei events.EventManager
+	om operations.Manager
 }
 
 func (bc *boundCallbacks) BlockchainOpUpdate(operationID *fftypes.UUID, txState blockchain.TransactionStatus, blockchainTXID, errorMessage string, opOutput fftypes.JSONObject) error {
-	return bc.ei.OperationUpdate(bc.bi, operationID, txState, blockchainTXID, errorMessage, opOutput)
+	return bc.om.SubmitOperationUpdate(&operations.OperationUpdate{
+		ID:             operationID,
+		State:          txState,
+		BlockchainTXID: blockchainTXID,
+		ErrorMessage:   errorMessage,
+		Output:         opOutput,
+	})
 }
 
 func (bc *boundCallbacks) TokenOpUpdate(plugin tokens.Plugin, operationID *fftypes.UUID, txState fftypes.OpStatus, blockchainTXID, errorMessage string, opOutput fftypes.JSONObject) error {
```

Comment:
Looks like the `plugin` parameter is now ignored/not needed?

Reply:
I've added it in for consistency with the tokens path 👍